src/util/transactions/index.ts

```diff
@@ -111,16 +113,17 @@ export async function getTransactionCallWeight({
       case TransactionTypes.REMARK:
         call = (await targetApi.tx.system.remark(remarkInput)).toU8a();
         // TODO [#121] Figure out what the extra bytes are about
-        call = call.slice(2);
+        //  call = call.slice(2);
```

Comment:
It isn't needed?

Reply:
btw, the last two byes that we slice exist because scale adds it. Not sure why, but we have detected that with other js developers. At the moment i will recommend to keep the `slice`